tensorflow/python/data/experimental/kernel_tests/optimize_dataset_test.py

```diff
@@ -674,6 +676,100 @@ def build_dataset():
 
     self.run_core_tests(build_dataset, 20)
 
+class ChooseFastestBranchDatasetCheckpointTest(
```

Comment:
Please move this to tensorflow/python/data/experimental/kernel_tests/optimization/choose_fastest_branch_dataset_test.py